ui/src/routes/_layout/organizations/$orgId/index.tsx

```diff
@@ -227,13 +227,13 @@ const OrganizationComponent = () => {
             table={table}
             setCurrentPageOptions={(currentPage) => {
               return {
-                ...Route,
+                to: Route.to,
```

Comment:
Just to make sure, you did try that this works on all pages without passing `params`? I have no doubt that it doesn't work on the organization page at `/`, but does it work for products, repos and runs as well, as those pages do have params?

Reply:
Yes, I tried it on all pages, it works without params.